corehq/apps/locations/models.py

```diff
@@ -228,7 +228,7 @@ class SQLLocation(SyncSQLToCouchMixin, MPTTModel):
     location_type = models.ForeignKey(LocationType)
     site_code = models.CharField(max_length=255)
     external_id = models.CharField(max_length=255, null=True)
-    metadata = json_field.JSONField(default={})
+    metadata = jsonfield.JSONField(default={})
```

Comment:
it looks like the library supports both callables and values as defaults: https://github.com/bradjasper/django-jsonfield/blob/master/jsonfield/fields.py#L147